integration_tests/commands/json_test.go

```diff
@@ -841,3 +840,88 @@ func TestJsonARRAPPEND(t *testing.T) {
 		})
 	}
 }
+func convertToArray(input string) []string {
+	input = strings.Trim(input, `"[`)
+	input = strings.Trim(input, `]"`)
+	elements := strings.Split(input, ",")
+	for i, element := range elements {
+		elements[i] = strings.TrimSpace(element)
+	}
+	return elements
+}
+func TestJSONNumIncrBy(t *testing.T) {
```

Comment:
Can you also add logic to validate the final state of the json values using `JSON.GET`? It will ensure the changes are being applied to the actual values correctly.